### ISSUE TITLE
Using chalk in AVA tests does not colorize the strings

### ISSUE THREAD

**Issue by KnisterPeter** [avajs/ava#1124]
I noticed that the following code does work as expected

```js
const chalk = require('chalk');
console.log(chalk.blue('hello world')); // as expected the result is blue
```

but this doesn't

```js
const test = require('ava');
const chalk = require('chalk');
console.log(chalk.blue('hello world')); // NOT expected the result is grey
test(t => t.fail());
```

I'm using ava 0.17.0 and chalk 1.1.3.

**Comment by sindresorhus** [comment]
// @thinkimlazy

**Comment by thinkimlazy** [comment]
Ye that what I meant. In my terminal test files with chalk disabled, but my terminal supports color. Sindre got colors.
I'll dig into
#1104

**Comment by novemberborn** [comment]
@sindresorhus @thinkimlazy what is the actual bug here? Is it AVA overriding the `chalk` settings on a global basis? That seems like an issue with `chalk`s design more than anything.

**Comment by unional** [comment]
I figure out the problem. It's in the `ansi-styles`. This issue happens on windows and `ansi-styles` does not produce the correct string in that environment.

Windows follow this (up to Windows 8):
http://www.termsys.demon.co.uk/vtansi.htm

**Comment by unional** [comment]
This is what need to be done on windows:
https://github.com/unional/aurelia-logging-color/blob/master/src/AnsiBrush.ts#L31

**Comment by Qix-** [comment]
@unional We do literally the exact same thing. Nobody else has reported any issues on Windows until this bug. There's no difference here. This is probably a configuration issue happening inside of AVA.

**Comment by unional** [comment]
Mentioned on the `ansi-styles` issue. It is not the same.

To see the difference: 

```sh
npm i chalk ava aurelia-logging aurelia-logging-color
```
```js
// index.js
var chalk = require('chalk')
var logging = require('aurelia-logging')
var color = require('aurelia-logging-color')
var test = require('ava')

logging.setLevel(logging.logLevel.debug)
var log = logging.getLogger('some log')
logging.addAppender(new color.ColorAppender())

test(_t => {
  console.log(chalk.bgGreen('suppose to be green'))
  log.debug('log something')
})
```
```sh
ava index.js
```

**Comment by Qix-** [comment]
It is the same. Check `supports-color` level reports, here. Nobody has reported anything on windows other than in this ticket. This is an AVA issue until I see definitive reasoning for it being an ansi-styles problem.

**Comment by unional** [comment]
Ok. You are correct. I do see that `supports-color` turn `chalk` off in `win32` environment.

When I worked on `aurelia-logging-color`, I can't use `ansi-style` because after bundling all dependencies the library become quite huge, and I recall seeing the color not rendered correctly.  That's why I went in and go through the code and the material to figure out how to get it working in my case.

However, right now I can't reproduce the problem. Everything seems to be working fine for `chalk`, `ansi-style` and so on. I must be dreaming back then. 😛 

As for the OP, I tried `chalk.enabled = true` to force coloring and it is working. FYI. 🌷 

```js
var chalk = require('chalk')
var test = require('ava')

chalk.enabled = true

test(_t => {
  console.log(`bgBlue: ${chalk.bgBlue('bgBlue')} and yellow: ${chalk.yellow('yellow')}`)
  console.log(chalk.bgCyan('cyan'), chalk.bgGreen('green'))
})
```
![image](https://cloud.githubusercontent.com/assets/3254987/22642601/36c1dc00-ec10-11e6-829e-57fb5c485524.png)
